sherpa/fit.py

```diff
@@ -482,7 +510,8 @@ def _get_callback(self, outfile=None, clobber=False):
             warning(e)
 
         self._dep, self._staterror, self._syserror = self.data.to_fit(
-            self.stat.calc_staterror)
+	    self.stat.calc_staterror)
+        self.bkg, junk1, junk2 = self.get_bkg_data( self._dep )
```

Comment:
Can this be changed to

    self.bkg, _, _ = self.get_bkg_data(self._dep)

to be explicit that we do not care about the second and third items returned by the call?